doc/cli/operator-sdk_migrate.md

```diff
@@ -4,7 +4,8 @@ Adds source code to an operator
 
 ### Synopsis
 
-operator-sdk migrate adds a main.go source file and any associated source files for an operator that is not of the "go" type.
+operator-sdk migrate adds a main.go source file and any associated source files
+				for an operator that is not of the "go" type.
```

Comment:
Same here:
```suggestion
for an operator that is not of the "go" type.
```

Reply:
The above comment was not solved. 
However, @hasbro17 and @ this doc is gen by the tool automatically. 

The error is here: (The string should be split and contact with "+".)

https://github.com/operator-framework/operator-sdk/pull/2373/files#diff-f43f447059b5702fca9bbe541e7818caR42

Reply:
thank you, it is done